plugins/search-backend-module-stack-overflow/api-report.md

```diff
@@ -0,0 +1,56 @@
+## API Report File for "@backstage/plugin-search-backend-module-stack-overflow"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+/// <reference types="node" />
+
+import { Config } from '@backstage/config';
+import { DocumentCollatorFactory } from '@backstage/plugin-search-common';
+import { IndexableDocument } from '@backstage/plugin-search-common';
+import { Logger } from 'winston';
+import { Readable } from 'stream';
+
+// @public
+export interface StackOverflowDocument extends IndexableDocument {
```

Comment:
This one is interesting. The frontend [also wants this type](https://github.com/backstage/backstage/blob/da6d238dd434f2a4f9dc5feaf5ca147ad4461df3/plugins/stack-overflow/src/search/StackOverflowSearchResultListItem/StackOverflowSearchResultListItem.tsx#L38) if it could, so it could even go in some form of `@backstage/plugin-search-backend-module-stack-overflow-common` package or so. But let's not worry about that right now.